util/mkerr.pl

```diff
@@ -500,6 +502,8 @@ sub help
     }
 
     print OUT "\n/*\n * $lib function codes.\n */\n";
+    print OUT "# ifdef OPENSSL_API_3\n";
```

Comment:
Why is that guard necessary?  ...  Ah, because `#include <openssl/opensslconf.h>` isn't always present.

May I suggest making sure that `#include <openssl/openssl.h>` always be present?

Reply:
okay, done.  revised commits pushed.